crates/lib3h/src/dht/rrdht.rs

```diff
@@ -20,8 +19,8 @@ impl RrDht {
         RrDht {
             inbox: VecDeque::new(),
             this_peer: PeerData {
-                peer_address: "FIXME".to_string(),
-                peer_uri: Url::parse("fixme://host:123").unwrap(),
+                peer_name: Lib3hUri::with_undefined("FIXME"),
```

Comment:
ditto, shouldn't be undefined.

Reply:
We shouldn't allow `PeerData::new()` The only reason we have it is for unit-testing.